src/components/GlobalStyles/GlobalStyles.js

```diff
@@ -1,6 +1,82 @@
 import { createGlobalStyle } from "styled-components";
 
 const GlobalStyle = createGlobalStyle`
+  :root {
+
+    /* Colors */
+    --theme-color--purple: #C3AFC9;
```

Comment:
Something I find tricky when starting a new project and assigning colours to variables is having enough flexibility to add more colours when they are inevitably added.

If you were to look at a lot of frameworks you'll see they use a system where they suffix colour variable with numbers to allow a spread of colours from light to dark across each colour e.g. red-100, red-200 etc and I can see you've used this on the light-grey colour below.

It may be worth applying the same suffix to the other colours so you have the flexibility to add lighter or darker versions of each colour.

There is a project called [open color](https://yeun.github.io/open-color/) to give you an example where they try to cover off a wide range of colours that may be required, using these specific versions is not required for your project but it is an example where perhaps instead of creating rose, wheat, peach etc colour variables you could use yellow-0, red-1, pink-2. This reduces the need to try to constantly come up with variable names.